inspirehep/records/marshmallow/base.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+from copy import deepcopy
+
+from inspire_dojson.utils import get_recid_from_ref, strip_empty_values
+from marshmallow import post_dump
+from marshmallow.schema import Schema
+
+
+class InspireDummyIncludeAllFieldsSchema(Schema):
+    """Ugly way to easily dump whole records without specifying every field manually"""
+
+    @post_dump(pass_original=True)
+    def include_original_fields(self, object, original_data):
+        for key, value in original_data.items():
+            if key not in object and key not in self.exclude:
+                object[key] = original_data[key]
+        object = self.populate_recid_from_ref(object)
+        return strip_empty_values(object)
+
+    def populate_recid_from_ref(self, record):
```

Comment:
this should be part of the other serializers as well on inspire-next is for all records (including ``lit`` and ``aut``) ☺️

Reply:
Yes, it should, but other serializes were never a part of this task... There should be separate task which would make this also available for literature and authors.